hedera-node/src/test/java/com/hedera/services/queries/token/TokenAnswersTest.java

```diff
@@ -29,17 +29,19 @@
 class TokenAnswersTest {
 	GetTokenInfoAnswer tokenInfo;
 	GetTokenNftInfoAnswer nftInfo;
+	GetAccountNftInfosAnswer accountNftInfo;
```

Comment:
Can we rename this one to accountNftInfos?

Reply:
sure